src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -69,7 +69,11 @@ public function provideDefaultOgPermissions(PermissionEventInterface $event) {
    *   The default role event.
    */
   public function provideDefaultRoles(DefaultRoleEventInterface $event) {
-    $event->addRole(OgRoleInterface::ADMINISTRATOR, ['label' => 'Administrator']);
+    $event->addRole([
```

Comment:
OK! I get your point. This seems like a good plan. I wanted to make it "easy" to implement a subscriber, but your idea makes sense. Is it acceptable for you to change this in a followup?